### DISCUSSION TITLE
Move multiple devices to a different rack

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#14685] [Help Wanted!] [answered]
When editing multiple devices at once from the `/dcim/devices/edit/` page, I see an option to change the Site or Location, but there is no way to change the Rack.  Is there a reason for this?
<img width="869" alt="image" src="https://github.com/netbox-community/netbox/assets/3991046/dc5629a7-c2b5-4181-8a2b-e1cd220dfcdc">

**Comment by candlerb** [comment, marked as answer]
I guess it could be added, although already-racked devices are likely to be problematic - it'll only work if the corresponding position/face in the new rack is free.

It would be fine for non-racked devices, i.e. those associated with a rack but have no rack position or face.  A bulk edit option for "Remove from rack", i.e. "Set Null" on rack position and face, and/or set the rack to null, would also be OK.

I haven't tested what happens if you try to bulk-edit the Site or Location of a device where that device has already been associated with a rack. Is it rejected, or does the device become disassociated from its previous rack (and rack position, if set)?

**Reply by llamafilm** [reply]
Good point, rack positions would be complicated.  I was thinking of unracked devices, but this only came up while I was investigating #14684.  I don't really have a strong use case for it.

**Reply by skvedo** [reply]
Just clearing out face and position would be sufficient.
It is easy then set, just, position one by one or even use reorder plugin.
https://github.com/netbox-community/netbox-reorder-rack
